phpcs.xml.dist

```diff
@@ -49,8 +49,7 @@
 	<arg name="parallel" value="8"/>
 	<!-- Limit to PHP files -->
 	<arg name="extensions" value="php"/>
-
-	<config name="ignore_warnings_on_exit" value="true"/>
+	<arg value="s"/>
```

Comment:
Show sniff codes in the report; useful if you need to ignore a particular line but don't remember the sniff name.